structslop_test.go

```diff
@@ -27,14 +32,45 @@ func Test(t *testing.T) {
 	analysistest.Run(t, testdata, structslop.Analyzer, "struct")
 }
 
+func TestApply(t *testing.T) {
+	dir := strings.Join([]string{".", "testdata", "src"}, string(os.PathSeparator))
```

Comment:
filepath.Join(...) instead of strings.Join

Reply:
This is intended. `filepath.Join` will call `Clean` before output, so relative path is cleaned.